maquetado/scss/detail.scss

```diff
@@ -64,10 +68,18 @@
   margin-top: 10px;
 }
 
+.logout {
+  margin-left: auto;
+}
+
 .margin-spaced-top {
   margin-top: 40px;
 }
 
+.mid-spaced-top {
+  padding-top: 100px;
```

Comment:
if the navbar changes its width this will break, you should use the navbar-height to calculate this value